components/setting_item_max.jsx

```diff
@@ -10,29 +10,117 @@ import SaveButton from 'components/save_button.jsx';
 import Constants from 'utils/constants.jsx';
 import * as Utils from 'utils/utils.jsx';
 
-export default class SettingItemMax extends React.Component {
-    constructor(props) {
-        super(props);
+export default class SettingItemMax extends React.PureComponent {
+    static defaultProps = {
+        infoPosition: 'bottom',
+        saving: false,
+        section: ''
+    };
+
+    static propTypes = {
+
+        /**
+         * Array of inputs selection
+         */
+        inputs: PropTypes.array,
+
+        /**
+         * Client error
+         */
+        client_error: PropTypes.string,
```

Comment:
Can we change this to be clientError?

Reply:
Noted. In addition,
- Changed server_error to serverError
- Updated handleSubmit's parameter of EmailNotificationSetting